recipes-core/packagegroups/packagegroup-westeros.bb

```diff
@@ -25,4 +25,4 @@ WESTEROS_SOC_rpi = "westeros-soc-drm"
 WESTEROS_SOC_rpi = "${@bb.utils.contains("MACHINE_FEATURES", "vc4graphics", "westeros-soc-drm", "westeros-soc-rpi", d)}"
 
 WESTEROS_RENDERER ?= ""
-WESTEROS_SOC_rpi = "${@bb.utils.contains("MACHINE_FEATURES", "vc4graphics", "", "westeros-render-dispmanx", d)}"
+#WESTEROS_SOC_rpi = "${@bb.utils.contains("MACHINE_FEATURES", "vc4graphics", "", "westeros-render-dispmanx", d)}"
```

Comment:
WESTEROS_SOC_rpi = "${@bb.utils.contains("MACHINE_FEATURES", "vc4graphics", "westeros-soc-drm", "westeros-render-dispmanx", d)}" should work for both vc4graphics and userland.